meb/core/train_eval.py

```diff
@@ -254,6 +258,10 @@ def train_model(
                     self.printer.print_train_test_validation(
                         train_metrics, test_metrics, epoch
                     )
+            if self.cf.weights_name:
+                torch.save(self.model.state_dict(), self.cf.weights_name + '_' + split_name + '.pth')

```

Comment:
This is a good feature update, but need to improve it further for the future.

Instead of weights_name, we should make a folder with a given name (+ current time to avoid duplicates) where the models are saved according to each split.

Additionally, we can make this a function that could be more easily overwritten if users want custom model saving, (e.g. LoRAs).

Reply:
adding the first part to commits. 
the second part I am not sure, because they can customize by changing the way they name the weights. and I believe it falls out of the context of this PR